From 0e08926b85b3357ebfc41828e15705a4f0bd5033 Mon Sep 17 00:00:00 2001
From: Jon Nettleton <jon.nettleton@gmail.com>
Date: Tue, 9 Sep 2014 11:11:55 +0200
Subject: [PATCH 1200/1201] ASOC: fsl_ssi: remove DMA_RESIDUE flag

This has been fixed in the SDMA driver so this flag is no longer needed.
---
 sound/soc/fsl/fsl_ssi.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/fsl/fsl_ssi.c b/sound/soc/fsl/fsl_ssi.c
index 9be24f2..049ba2e 100644
--- a/sound/soc/fsl/fsl_ssi.c
+++ b/sound/soc/fsl/fsl_ssi.c
@@ -1551,7 +1551,7 @@ static int fsl_ssi_probe(struct platform_device *pdev)
 			if (ret)
 				goto error_pcm;
 		} else {
-			ret = imx_pcm_dma_init(pdev, SND_DMAENGINE_PCM_FLAG_NO_RESIDUE,
+			ret = imx_pcm_dma_init(pdev, NULL,
 						IMX_SSI_DMABUF_SIZE);
 			if (ret)
 				goto error_pcm;
-- 
1.7.9.5

